src/Root/App/index.tsx

```diff
@@ -23,11 +23,14 @@ async function digestMessage(message: string) {
 
 function App() {
     const [loggedIn, setLoggedIn] = useStoredState<string>(
-        'dfid-login',
+        'bek-login',
         'false',
     );
     const [password, setPassword] = useState('');
-    const [administrator, setAdministrator] = useState(false);
+    const [administrator, setAdministrator] = useStoredState<boolean>(
+        'bek-admin',
+        false,
```

Comment:
As localStorage doesn't support boolean, this needs to be a string not boolean, similar to loggedIn.